File: Atividades Udemy/Java/secao06/atividade06.java
package secao06;
import java.util.Scanner;
public class atividade06 {

	public static void main(String[] args) {
		 int c;
		 float n, e = 0, salario, valor_hora = (float) 10.00;
		 Scanner teclado = new Scanner(System.in);
		 
		 System.out.println("Informe o código: ");
		 c = teclado.nextInt();
		 System.out.println("Informe a quantidade de horas trabalhadas: ");
		 n = teclado.nextFloat();
		 
		 if(n > 50) {
			 e = (float)(n - 50) * (float)20.00;
			 salario = (50 * valor_hora) + e;
			 System.out.printf("Salário Total %.2f\n", salario);
			 System.out.printf("Salário excedente %.2f\n", e);
		 } else {
			 salario = n * valor_hora;
			 System.out.printf("Salário Total %.2f\n", salario);
			 System.out.printf("Salário excedente %.2f\n", e);
		 }
		 teclado.close();
	}

}

File: Atividades Udemy/Python/secao06/atividade06.py
c = int(input("Informe o código: "))
e = 0
n = int(input("Informe a quantidade de horas trabalhadas: "))

if(n > 50):
    e = n - 50
    n = n - e
    extra = n * 20.00
    salario = n * 10.00
    salariototal = salario + extra
    print(f"Salário: R$ {salario:.2f}")
    print(f"Extra: R$ {extra:.2f}")
    print(f"Salário total: R$ {salariototal:.2f}")
else: 
    salario = n * 10.00
    print(f"Salário: R$ {salario:.2f}")